src/mavedb/view_models/target_sequence.py

```diff
@@ -40,14 +44,14 @@ def validate_identifier(cls, field_value, values, field, config):
         return field_value
 
     @validator("label")
-    def check_alphanumeric(cls, field_value, values, field, config) -> str:
+    def label_does_not_include_colon(cls, field_value, values, field, config) -> str:
         if isinstance(field_value, str):
-            is_alphanumeric = field_value.replace("_", "").isalnum()
-            if not is_alphanumeric:
-                raise ValidationError(
-                    f"Target sequence label `{field_value}` can contain only letters, numbers, and underscores."
-                )
-        return field_value
+            if ":" in field_value:
+                raise ValidationError(f"Target sequence label `{field_value}` may not contain a colon.")
+
+        # Sanitize the label by stripping leading/trailing whitespace and replacing any internal whitespace with
+        # underscores. Fully qualified variants should never contain whitespace.
+        return sanitize_target_sequence_label(field_value)
```

Comment:
This is getting called when field_value is None, leading to an unhandled exception.